rsocket-core/src/main/java/io/rsocket/lease/Lease.java

```diff
@@ -69,4 +69,9 @@ default boolean isExpired() {
   default boolean isExpired(long now) {
     return now > expiry();
   }
+
+  /** Checks if the lease has not expired and there are allowed requests available */
```

Comment:
nit: gradle complains about the missing @return

Reply:
I will update javadocs with detailed explanations once apis and implementations are agreed